pennylane_qiskit/qiskit_device.py

```diff
@@ -181,15 +180,24 @@ def process_kwargs(self, kwargs):
         self.run_args.update(kwargs)
 
     def set_transpile_args(self, **kwargs):
-        """The transpile argument setter."""
+        """The transpile argument setter.
+
+        Keyword Args:
+            kwargs (dict): keyword arguments to be set for the Qiskit transpiler. For more details,
+                see the `Qiskit documentation <https://qiskit.org/documentation/stubs/qiskit.compiler.transpile.html>`_.
```

Comment:
As with the example in `pennylane_qiskit/ibmq.py`, instead of a general item, we'd want some specific set of keyword arguments here, right?

Reply:
@antalszava Yes that would be ideal but they are so many transpiler arguments that I thought it was better just to link it, what do you think?